### PR TITLE
MAT-6796: bugfix cmsID when 0

### PR DESCRIPTION
**Pull request by mcmcphillips**
## PR Template

Jira Ticket: [MAT-6796](https://jira.cms.gov/browse/MAT-6796)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x]  Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/app.ts [hunk 1]
```diff
@@ -62,7 +63,9 @@ export const lambdaHandler = async (event: S3Event): Promise<Measure> => {
       logAndMail(`Measure name: ${matMeasure?.manageMeasureDetailModel.measureName}`);
       logAndMail("Measure version in MADiE: 0.0.000");
       logAndMail(`Measure cqlLibraryName: ${matMeasure?.manageMeasureDetailModel.cqllibraryName}`);
-      logAndMail(`CMS ID: ${matMeasure?.manageMeasureDetailModel.eMeasureId}`);
+      if (cmsId) {
+        logAndMail(`CMS ID: ${cmsId}`);
```
Comment (nmorasb) [inline lambda-function/src/app.ts:67]: Can we add an extra test to bring the PR coverage up for when the cmsId is non-zero? (Coverage is reporting this line as unhit)

Comment (mcmcphillips) [reply lambda-function/src/app.ts:67]: should be added now.

**Review by nmorasb — APPROVED** [review]
